crates/chronicle/src/bootstrap/cli.rs

```diff
@@ -1054,6 +1054,10 @@ impl SubCommand for CliModel {
                             .env("USERINFO_URI")
                             .help("URI of the OIDC UserInfo endpoint")
                     }
+                    ).group(
+                        ArgGroup::new("oidc-endpoint-address")
+                            .args(&["jwks-address", "userinfo-address"])
+                            .multiple(true)
```

Comment:
Got it. And this means if something `requires` `"oidc-endpoint-address"`, then they must provide one of the members of the `ArgGroup`.